Tools/AP_Periph/can.cpp

```diff
@@ -902,7 +902,23 @@ static void process1HzTasks(uint64_t timestamp_usec)
 
 #if !defined(HAL_NO_FLASH_SUPPORT) && !defined(HAL_NO_ROMFS_SUPPORT)
     if (periph.g.flash_bootloader.get()) {
+        const uint8_t flash_bl = periph.g.flash_bootloader.get();
         periph.g.flash_bootloader.set_and_save_ifchanged(0);
+        if (flash_bl == 42) {
```

Comment:
This stuff should probably be wrapped in # ifdef debug

Reply:
I want it available in production boards, so I can ask an end user to test it if need be